src/codeu/chat/client/ClientUser.java

```diff
@@ -123,7 +131,7 @@ public boolean addUser(String name, String password) {
   }
 
   public void showAllUsers() {
-    updateUsers();
+    //updateUsers();
```

Comment:
Why is this commented out? Should it be deleted? Has the functionality been moved?